gtk4/src/builder_rust_scope.rs

```diff
@@ -0,0 +1,143 @@
+use crate::subclass::builder_scope::BuilderScopeImpl;
+use crate::Builder;
+use crate::BuilderClosureFlags;
+use crate::BuilderError;
+use crate::BuilderScope;
+use glib::prelude::*;
+use glib::subclass::prelude::*;
```

Comment:
```suggestion
use crate::prelude::*;
```

the glib subclass's prelude is included in the gtk (crate) one